app/models/job_template_effective_user.rb

```diff
@@ -4,8 +4,6 @@ class JobTemplateEffectiveUser < ActiveRecord::Base
 
   before_validation :set_defaults
 
-  belongs_to :job_template
```

Comment:
Becuase it was there twice, so I left there just one https://github.com/iNecas/foreman_remote_execution/blob/katello-integration/app/models/job_template_effective_user.rb#L3